contracts/AccessTokenVerifier.sol

```diff
@@ -21,12 +22,12 @@ contract AccessTokenVerifier is IAccessTokenVerifier, KeyInfrastructure {
     // solhint-disable var-name-mixedcase
     bytes32 public DOMAIN_SEPARATOR;
 
-    constructor(address root) KeyInfrastructure(root) {
+    constructor(uint256 chainId, address root) KeyInfrastructure(root) {
```

Comment:
Aha, brilliant idea! I reverted on this addition to the 0.8 version because solidity doesn't support it natively, but yes, assembly does give us access to this variable! Adding this back :D